src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -540,28 +577,14 @@ TEST_CASE(
   trrm.schedule_.push_back(std::chrono::milliseconds(12));
   trrm.schedule_.push_back(std::chrono::milliseconds(16));
   trrm.gen_duration_ = std::make_unique<std::chrono::nanoseconds>(16000000);
+  trrm.distribution_ = std::uniform_int_distribution<uint64_t>(
+      0, mdl->GetDataStreamsCount() - 1);
```

Comment:
The distribution needs to be set here because the code is written in a way that the default distribution ranges from 0 to like INT_MAX if we don't actually supply a real JSON input file (we're mocking it with a string). It actually needs to be 0 to the number of data streams in the "input json" (string).